internal/api/utils.go

```diff
@@ -16,6 +16,13 @@ import (
 	"github.com/redhatinsights/ros-ocp-backend/internal/logging"
 )
 
+var NotificationsToShow = map[string]string{
```

Comment:
This just feels does not belong to utils, as per what goes where /ros-ocp-backend/blob/main/internal/types/kruizePayload/common.go feels better way to keep this?

Reply:
I have moved the constant and some other structs to a dedicated `common.go` file for the api package

Reply:
I have moved the constant and some other structs to a dedicated `common.go` file for the api package